File: scratch/restart_pm2.py
import paramiko, sys
c = paramiko.SSHClient()
c.set_missing_host_key_policy(paramiko.AutoAddPolicy())
c.connect('65.20.85.75', 22, 'graceandforce', 'wvpi2!ZnTcV];ncy')
def run(cmd):
    print(f"\n>>> {cmd}")
    stdin, stdout, stderr = c.exec_command('export NVM_DIR="$HOME/.nvm"; [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"; ' + cmd)
    for line in iter(stdout.readline, ""): sys.stdout.buffer.write(line.encode('utf-8'))
    for line in iter(stderr.readline, ""): sys.stdout.buffer.write(line.encode('utf-8'))
run('pm2 restart grace-api')
run('pm2 logs grace-api --lines 20 --nostream')
c.close()

File: restart_pm2.js
const { Client } = require('ssh2');
const conn = new Client();
conn.on('ready', () => {
  conn.exec('source ~/.nvm/nvm.sh && pm2 restart grace-api', (err, stream) => {
    if (err) throw err;
    stream.on('data', d => process.stdout.write(d))
          .stderr.on('data', d => process.stderr.write(d));
    stream.on('close', () => conn.end());
  });
}).connect({ host: '65.20.85.75', port: 22, username: 'graceandforce', password: 'wvpi2!ZnTcV];ncy' });
